blueprint/index.md

```diff
@@ -1,224 +1,250 @@
+2. Navigate to **App registrations**, and click **New registration**.
 
-![New Registration for an Azure app](images/0ACreateNewAzureAppRegistration.png)
+   ![New Registration for an Azure app](images/0ACreateNewAzureAppRegistration.png)
 
-2. Name the new application and select the preferred supported account type. Any account type works for a single Genesys Cloud organization.
+3. Enter a name for the application and select the preferred supported account type. Any account type works for a single Genesys Cloud organization. Click **Register**.
 
-![Register the new Azure app](images/0BRegisterNewApp.png)
+   ![Register the new Azure app](images/0BRegisterNewApp.png)
 
-3. View the new application. Copy the Application and Directory IDs for later use.
+4. On the application's Overview page, copy **Application (client) ID** and **Directory (tenant) ID** for later use.
+   
+   ![Newly registered Azure app](images/0CNewApp.png)
 
-![Newly registered Azure app](images/0CNewApp.png)
+5. Navigate to **Certificates & secrets** and click **New client secret**.
+   
+   ![Create client secret](images/0DCreateClientSecret.png)
 
-4. Navigate to **Certificates & Secrets** and click **+ New Client Secret**.
+6. Copy the client secret value for later use.
 
-![Create client secret](images/0DCreateClientSecret.png)
+   **Note:** You cannot view or retrieve the client secret value later.
 
-5. Copy the client secret value for later use.
-  **Warning:** You cannot view or retrieve the client secret value later.
+   ![Copy and save client secret](images/0ECopyClientSecret.png)
 
-![Copy and save client secret](images/0ECopyClientSecret.png)
-6. Confirm that your Genesys Cloud permission is created in Azure.
-![Microsoft Azure Permissions](images/azurePerm.png)
+7. Check your Genesys Cloud permission created in Azure.
+   
+   ![Microsoft Azure Permissions](images/azurePerm.png)
 
-## Configure Genesys Cloud
-### Create a Web Services Data Actions Integration
-To enable communication from Genesys Cloud to Microsoft Azure and Microsoft Teams, you need to create a Web Services Data Action integration and provide it with the proper fields and values:
+### Configure Genesys Cloud
 
-1. Search for and install the **Web Services Data Actions** integration from within Genesys Cloud or from the AppFoundry. For more information, see [About the data actions integrations](https://help.mypurecloud.com/?p=209478 "Opens the Data Actions overview article").
+### Add a web services data actions integration
+To enable communication from Genesys Cloud to Microsoft Azure and Microsoft Teams, you must add a web services data actions integration:
 
-![Web Services Data Actions in Genesys Cloud Integrations Search](images/1AWebServicesDataActionInstall.png)
+1. Install the **Web Services Data Actions** integration from Genesys Cloud or from the AppFoundry. See [About the data actions integrations](https://help.mypurecloud.com/?p=209478 "Opens the Data Actions overview article").
```

Comment:
I searched for "Web Services Data Actions integration" in the App Foundry and could not find it. Please verify this step.  https://appfoundry.genesys.com/

If this is not a pre-packed/properly named integration, then do not capitalize references to it. "Install a "web services data actions integration..."